providers/ethereum-provider/src/index.ts

```diff
@@ -229,11 +223,19 @@ class EthereumProvider implements IEthereumProvider {
     return Number(chainId.split(":")[1]);
   }
 
-  private setChainId(chains: string[]) {
```

Comment:
I'm confused about the name change here

Reply:
There are 2 methods `setChainId` and `setChainIds` now. One takes in array, the other single chain (string) and both perform different logic. 

`ethereum-provider` is not single chain, but it it is single namespace `eip155`